_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
-NewPipe notifies you about new versions. You can download them when you press the notification. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository. If you have problems installing you may need to uninstall NewPipe then reinstall (make sure to backup data).
+NewPipe notifies you of new versions in a notification you can click to download it. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), you will be notified of all app news, including [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/). Please keep in mind it can take F-Droid a while to update their repository. If you have problems installing you may need to uninstall NewPipe, and then reinstall it. (Make sure to back up you data.)
 
 If you already had NewPipe installed through F-Droid's repository, to get this version of NewPipe you can do one of the following:
-- Wait for them to update or
-- Switch to the NewPipe repository by following the [directions in the announcement]({% post_url 2020-05-30-f-droid-repo %}) (if you had previously installed NewPipe from GitHub releases you will not have to uninstall NewPipe to switch)
+- Wait for F-Droid to ship the new NewPipe
+- Switch to the NewPipe repository in F-Droid by following the [directions in the announcement]({% post_url 2020-05-30-f-droid-repo %}) (no need to remove any NewPipe installed from GitHub releases first).
+
+Now that you are on NewPipe 0.20.0, please let us know what your experience is, especially bugs in need of fixing.
+As usual, you can reach out via IRC ([#newpipe on freenode](https://webchat.freenode.net/?channels=newpipe)), [open issues on GitHub](https://github.com/TeamNewPipe/NewPipe/issues/new) or, ideally, use our built-in crash reporter to send in machine-readable issue reports.
+You can even [send in fixes](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#bug-fixing) yourself.
+Feel free to ask other questions in the comments here, and someone will reply.
+
 
-Now that you've updated, please let us know what your experience of the latest release is, especially bugs in need of fixing. As usual, you can reach out to us via IRC ([#newpipe on freenode](https://webchat.freenode.net/?channels=newpipe)), [open issues on GitHub](https://github.com/TeamNewPipe/NewPipe/issues/new) or, ideally, use our built-in crash reporter to send us machine-readable issue reports. You can even [send in fixes](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#bug-fixing) yourself. If you have any other questions feel free to send them in the comments here and someone will reply to you.
```

Comment:
No, actually reading. Havent updated, because it is "upgrading". Who on IRC? Can I talk to the developers and community directly. That is kind of cool, why not mention that? Someone who?

Reply:
> No, actually reading the text at this point.

We're not being literal. XD

> Haven't updated, because it is "upgrading". 

Sure.

> Who on IRC?

"us", so the team behind the app, otherwise we would have mentioned the 3rd party or company that handles customer care for us.

> Someone who?

No one specific. Anyone who is online at that time.

Reply:
Vague, and more vague.